fahrschein/src/test/java/org/zalando/fahrschein/NakadiReaderDeserializationTest.java

```diff
@@ -184,11 +184,11 @@ public void shouldDeserializeBusinessEventMetadata() throws IOException {
         assertThat(metadata.getFlowId(), Matchers.equalTo("ABCD"));
         assertThat(metadata.getOccurredAt(), Matchers.equalTo(OffsetDateTime.of(2016, 10, 26, 19, 20, 21, 123_000_000, ZoneOffset.UTC)));
         assertThat(metadata.getReceivedAt(), Matchers.equalTo(OffsetDateTime.of(2016, 10, 26, 20, 21, 22, 0, ZoneOffset.ofHours(1))));
-        assertThat(metadata.any(), Matchers.equalTo(new HashMap<String, Object>()));
+        assertThat(metadata.getSpanCtx(), Matchers.equalTo(Collections.emptyMap()));
```

Comment:
Not really. Although the majority of open tracing libraries does return a null if no span context is present. I'd say that if you document that an empty map is returned in case of no span context in the metadata we can leave it at that.

Reply:
Actually, turns out that libraries are a bit more diverse than I thought. Some throw exception, others return a null. Returning an empty Map seems like a safe enough approach.